components/Activity/HeroCarousel.tsx

```
@@ -0,0 +1,192 @@
+import { CSSProperties, FC, useContext, useEffect, useState } from 'react';
+import { TextTruncate } from 'idea-react';
+import { TableCellLocation } from 'mobx-lark';
+import { Badge, Button, Card, Carousel, Col, Container, Row, Stack } from 'react-bootstrap';
+
+import { Activity, ActivityModel } from '../../models/Activity';
+import { I18nContext } from '../../models/Translation';
+import { LarkImage } from '../LarkImage';
+import styles from './HeroCarousel.module.less';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)